rpm/go-vendor-tools.toml

```diff
@@ -0,0 +1,18 @@
+[archive]
+
+[licensing]
+detector = "askalono"
+[[licensing.licenses]]
+path = "vendor/github.com/fsouza/go-dockerclient/DOCKER-LICENSE"
+sha256sum = "04649aa5a97550d0bb083955b37586eb0ed6c6caa6e8a32f9cc840bbb3274254"
+expression = "Apache-2.0"
+
+[[licensing.licenses]]
+path = "vendor/github.com/shirou/gopsutil/v3/LICENSE"
+sha256sum = "ad1e64b82c04fb2ee6bfe521bff01266971ffaa70500024d4ac767c6033aafb9"
+expression = "BSD-3-Clause"
+
+[[licensing.licenses]]
+path = "vendor/gopkg.in/yaml.v3/LICENSE"
+sha256sum = "d18f6323b71b0b768bb5e9616e36da390fbd39369a81807cca352de4e4e6aa0b"
+expression = "MIT"
```

Comment:
sorry I forgot to ask why this file is required? (I'm not familiar with the go-vendor tool)

Reply:
the go-vendor-tools macros process the license files during the rpm build process. The `toml` file provides configuration to that process. E.G. specify alternate license detector, set licenses for modules where the detector cannot find a license, exclude directories or files from license detection.

The `toml` file can also replace a module with a different version if needed. For example to resolve an urgent CVE.

See https://fedora.gitlab.io/sigs/go/go-vendor-tools/config/ for options.